platform/windows/detect.py

```diff
@@ -789,7 +789,8 @@ def configure_mingw(env: "SConsEnvironment"):
         if try_cmd("as --version", env["mingw_prefix"], env["arch"]):
             env["AS"] = mingw_bin_prefix + "as"
         if try_cmd("gcc-ar --version", env["mingw_prefix"], env["arch"]):
-            env["AR"] = mingw_bin_prefix + "gcc-ar"
+            if os.name != "nt":  # Don't affect cross-compilation.
```

Comment:
Probably should be something like this, since `ar` without prefix might be missing (and will always be missing when using `mingw_prefix`):

```
if os.name != "nt":
    if try_cmd("gcc-ar --version", env["mingw_prefix"], env["arch"]):
        env["AR"] = mingw_bin_prefix + "gcc-ar"
else:
    if try_cmd("ar --version", env["mingw_prefix"], env["arch"]):
        env["AR"] = mingw_bin_prefix + "ar"
```